web/angular-wallet/proxy.config.json

```diff
@@ -1,6 +1,6 @@
 {
   "/burst": {
-    "target": "https://wallet1.burst-team.us:2083/",
+    "target": "http://testnet.getburst.net:6876",
```

Comment:
Do I bother you with that changes?

Reply:
No I probably did this on accident while testing. I think it should be this way by default 👍